google-cloud-binary_authorization-v1beta1/lib/google/cloud/binary_authorization/v1beta1/binauthz_management_service_v1_beta1.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+require "gapic/common"
+require "gapic/config"
+require "gapic/config/method"
+
+require "google/cloud/binary_authorization/v1beta1/version"
+
+require "google/cloud/binary_authorization/v1beta1/binauthz_management_service_v1_beta1/credentials"
+require "google/cloud/binary_authorization/v1beta1/binauthz_management_service_v1_beta1/paths"
+require "google/cloud/binary_authorization/v1beta1/binauthz_management_service_v1_beta1/client"
+
+module Google
+  module Cloud
+    module BinaryAuthorization
+      module V1beta1
+        ##
+        # Google Cloud Management Service for Binary Authorization admission policies
+        # and attestation authorities.
+        #
+        # This API implements a REST model with the following objects:
+        #
+        # * {::Google::Cloud::BinaryAuthorization::V1beta1::Policy Policy}
+        # * {::Google::Cloud::BinaryAuthorization::V1beta1::Attestor Attestor}
+        #
+        # To load this service and instantiate a client:
+        #
+        #     require "google/cloud/binary_authorization/v1beta1/binauthz_management_service_v1_beta1"
+        #     client = ::Google::Cloud::BinaryAuthorization::V1beta1::BinauthzManagementServiceV1Beta1::Client.new
+        #
+        module BinauthzManagementServiceV1Beta1
```

Comment:
Do you want to remove `V1Beta1` from this `BinauthzManagementServiceV1Beta1` module name?

Reply:
Oh my, thanks for catching this. I need to pay attention and look for these when I generate these libraries.

Reply:
Fixed.